lib/rubocop/cop/layout/empty_line_between_defs.rb

```diff
@@ -89,6 +89,8 @@ class EmptyLineBetweenDefs < Base
         extend AutoCorrector
 
         MSG = 'Use empty lines between %<type>s definitions.'
+        MSG_RANGE = '%<min>d..%<max>d empty lines between %<type>s definitions expected; ' \
+                    'found %<actual>d.'
```

Comment:
I feel `between %<min> and %<max> (both inclusive)` would read better in a message than the range syntax.

Reply:
I originally had written it less terse but I think rubyists will understand range syntax without having to be overly verbose.